src/Symfony/Component/Form/FormEvents.php

```diff
@@ -11,6 +11,12 @@
 namespace Symfony\Component\Form;
 
 /**
+ * To learn more about how form events work check the documentation
+ * entry at {@link http://symfony.com/doc/current/components/form/form_events.html}
```

Comment:
I do like this type of linking, and if we're going to have it, then let's use Jordi's solution (use `/any` but, redirect to `/current` with a special message). I can't think of a better "UX", but if we could get an actual designer-type (or Javier, since he's very good at this) to think about how this would look/work, then I think it will be great :).

One added challenge is that I could imagine these links potentially being anchor links in some cases, so we may not be at the top of the page.